integrationTests/singleShard/unsignedTransaction/testInitializer.go

```diff
@@ -0,0 +1,248 @@
+package unsignedTransaction
```

Comment:
no need for this file. The 2 tests could stay in the same package with transactions.

Reply:
done